app/controllers/concerns/sufia/file_sets_controller_behavior.rb

```diff
@@ -29,6 +29,7 @@ module FileSetsControllerBehavior
 
     def new
       @upload_set_id = ActiveFedora::Noid::Service.new.mint
+      @presenter = Sufia::WorkShowPresenter.new(::SolrDocument.new, nil)
```

Comment:
Good point. I've updated the controller that calls this view to set the `@work_id` variable instead.